lib/aytests/image_builder.rb

```diff
@@ -22,6 +22,7 @@ class ImageBuilder
     SSH_ADDRESS = "127.0.0.1"
     SSH_PORT = "22"
     WEBSERVER_PORT = "8888"
+    MAC_ADDRESS = "02:00:00:12:34:56"
```

Comment:
You know what? I didn't realize that it was 123456 at the end. :smile: Too early.